packages/appstore/src/components/account/account.scss

```diff
@@ -48,40 +46,21 @@
             width: 6.9rem;
             height: 3.2rem;
             &--name {
-                height: 1.8rem;
-                font-style: normal;
-                font-weight: 700;
-                font-size: 1.2rem;
-                line-height: 1.8rem;
-                color: $color-black-1;
                 @include mobile {
+                    font-size: 1rem;
```

Comment:
The size and height and other props of `Text` can be changes depends of `isMobile() ` condition when using the Text component.

ex: <Tex size={isMobile() ? 'xxxs' : 's'}>...</Text>